mongodbatlas/data_source_mongodbatlas_event_trigger.go

```diff
@@ -127,7 +127,10 @@ func dataSourceMongoDBAtlasEventTrigger() *schema.Resource {
 }
 
 func dataSourceMongoDBAtlasEventTriggerRead(ctx context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
-	conn := meta.(*MongoDBClient).Realm
+	conn, err := meta.(*MongoDBClient).GetRealmClient(ctx)
```

Comment:
Can you run the tests of the resources that use the realm client and paste the result?

Reply:
Done, added the results in the description, thank you @thetonymaster